net-misc/unison/unison-2.51.4_rc2.ebuild

```diff
@@ -25,6 +25,8 @@ RDEPEND="gtk? ( dev-ml/lablgtk:2=
 	|| ( net-misc/x11-ssh-askpass net-misc/ssh-askpass-fullscreen ) )
 	>=app-eselect/eselect-unison-0.4"
 
+QA_FLAGS_IGNORED="/usr/bin/unison{,-fsmonitor}-2.51"
+
```

Comment:
```
 * QA Notice: Files built without respecting CFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/bin/unison-fsmonitor-2.51
 * /usr/bin/unison-2.51
 * QA Notice: Files built without respecting LDFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/bin/unison-2.51
 * /usr/bin/unison-fsmonitor-2.51
```
This doesn't seem to work though :I 

I can merge the cleanups only if you wish, but this'd seem pointless. Just let me know.

Reply:
Please disregard this PR ...

Reply:
Commit, or PR? I see 3 commits in here, where 2 could be merged (the cleanups)

Reply:
If you like those cleanups please use them.  I won't do further contributions to this package.

Reply:
Will do, thanks!